templates/inspec/plural_resource.erb

```diff
@@ -29,7 +29,7 @@ name = "google_#{product_ns.downcase}_#{object.name.underscore}"
   filter_table_config = FilterTable.create
 <% object.all_user_properties.each do |prop| -%>
 
-  <%= "filter_table_config.add(:#{prop.out_name.pluralize}, field: :#{prop.name})" -%>
+  <%= "filter_table_config.add(:#{prop.out_name.pluralize}, field: :#{prop.out_name})" -%>
```

Comment:
Why doesn't this break anything?

Reply:
The InSpec FilterTable allows users to reference a specific field within a list of hashes. It will grab the value of that field from each hash and return it as a list. This `config.add(name, field)` tells the FilterTable to allow users to reference the field at `field` from each hash by using the key `name`. 

Previously we were just storing the JSON representation of the resources that was retrieved directly from the API, and aliasing the out_name.pluralize to reference the name that the API returns. This worked for top level properties, but not for properties that are nested 2+ levels. 

Using the transformers we are now parsing each resource using the same logic as in singular resources, allowing us to store these using their proper `out_name` as actual ruby objects

Basically this just changes how the internals of plural resources parse and surface attributes, no user facing changes

Reply:
Awesome! I was worried that this changed the InSpec control API.